tests/cc51/input.dat

```diff
@@ -21,5 +21,5 @@ eom_ref = -76.0519836445 # TEST
 energy('eom-cc3')
 compare_values(scf_ref, variable("SCF TOTAL ENERGY"), 6, "SCF energy")              #TEST
 compare_values(cc3_ref, variable("CC3 TOTAL ENERGY"), 6, "CC3 energy")              #TEST
-compare_values(eom_ref, variable("CC ROOT 1 TOTAL ENERGY"), 6, "EOM-CC3 energy")    #TEST
+compare_values(eom_ref, variable("CC3 ROOT 1 TOTAL ENERGY"), 6, "EOM-CC3 energy")    #TEST
```

Comment:
it'd be nice to test both `CC ROOT` and `CC3 ROOT`. tddft, detci, and cc* generally have this broader label access. maybe just here (cc51) and cc45 will do. stdsuite will check this when it expands to excited state methods.

Reply:
I didn't intend for `CC ROOT` to continue working precisely because TDDFT does _not_ have this broader label access. Run the following input, and you will find many TD-B3LYP variables and no TD-DFT variables, just like it was before #2462. (I checked at the time.)
```
molecule h2o {
  O
  H 1 0.9 
  H 1 0.9 2 104.0
}

set {
  basis cc-pVDZ
  tdscf_states [1, 1, 1, 1]
}

wfn = energy('td-b3lyp', return_wfn=True)[1]

for key in wfn.variables():
    print(key)

for key in variables():
    print(key)
```

I can add such access to both TDDFT and EOM if you want, but that's new.

Reply:
You're right, I was probably thinking of DFT TOTAL ENERGY, not TDDFT TOTAL ENERGY. So for dft ground states, `<fctl> TOTAL ENERGY` is sparsely defined whereas `DFT TOTAL ENERGY` is assured. And for dft excited states, `TD-<fctl>` is assured and `TDDFT TOTAL ENERGY` not present. :-)

For here, I'd appreciate having the generic `CC` psivars back. Rationalizing the dft/tddft situation is a future matter.

Reply:
You're more familiar with this state of affairs, so please file an issue on the DFT psivar situation.

In the meantime, I can restore the `ROOT CC` psivars.

Reply:
> Rationalizing the dft/tddft situation is a future matter.

https://github.com/psi4/psi4/issues/2477